examples/with-next-sass/next.config.js

```diff
@@ -0,0 +1,4 @@
+const withSass = require('@zeit/next-sass')
+module.exports = withSass({
+  cssModules: true
```

Comment:
I think it's more common to not use css modules and use global styles instead. The other behavior should be mentioned in the readme.

Reply:
Modified. :)